engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2081,6 +2081,10 @@ public void updateRecoveryCheckpoints(PTOperator operator, UpdateCheckpointsCont
     if (checkpointGroup.size() > 1) {
       for (OperatorMeta om : checkpointGroup) {
         Collection<PTOperator> operators = plan.getAllOperators(om);
```

Comment:
It would be good to understand that, since it would make the extra code to find unifiers here unnecessary. It is possible that this is a result of changing how single unifiers are mapped in the physical plan in an earlier release. If so we should check if getAllOperators needs to be modified.